playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -261,20 +264,48 @@ def logs_enricher(event: PodEvent, params: LogEnricherParams):
             )
         return
 
+    containers: list[str] = []
+    all_statuses = pod.status.containerStatuses + pod.status.initContainerStatuses
+    if params.container_name:
+        containers = [params.container_name]
+    elif event.get_subject().container:
+        # support alerts with a container label, make sure its related to this pod.
+        containers = [
+            container_status.name
+            for container_status in all_statuses
+            if container_status.name == event.get_subject().container
+        ]
+
+    # If no container is specified, find containers in an unhealthy state. good for pending pods
+    if len(containers) == 0:
+        containers = [
+            container_status.name
+            for container_status in all_statuses
+            if container_status.state.waiting is not None and container_status.restartCount >= 1
```

Comment:
I think `logs_enricher` can be called not only for unhealthy pods.
Some users can call it on high cpu or memory alert.
I don't think we should filter out unhealthy containers. 

If `previous` is `true`, we should filter out containers that were not restarted.
WDYT?